ui/src/widgets/ui-gauge/UIGauge.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <component :is="`ui-${gtype}`" v-if="['gauge-tile', 'gauge-battery', 'gauge-tank'].includes(gtype)" :id="id" :props="dynamicProps" :value="value" />
-    <ui-gauge-dial v-else :id="id" :props="dynamicProps" :value="value" />
+    <ui-gauge-dial v-else :id="id" :key="updateGaugeDial" :props="dynamicProps" :value="value" />
```

Comment:
Why? I don't see any reason for this change?

Reply:
Gayan added this to ensure dynamic updates vai `ui_update` cause the gauge to update.

Scenario: User sends `{ ui_update: { gtype: "gauge-34" } }` (without `payload`) - gauge would not update until a later msg containing a payload arrived.

Reply:
The gauges didn't get scaled and rendered properly after changing dynamic properties therefore the best simple solution was to re-render the dial gauges on dynamic property changes